docs/guides/insar_product_guide_template.md

```diff
@@ -0,0 +1,183 @@
+
+{% block workflow %}{% endblock %}
+
+{% block packaging %}{% endblock %}
+
+## Limitations
+### Baseline Calculation
+The baseline is defined as the difference of the platform positions when a given area is imaged. HyP3 baselines are calculated using the best state vectors available. If precise orbits are not yet available for the input granules, restituted orbits will be used. The original predicted orbits are not used for InSAR processing in HyP3. **If no restituted or precise state vectors are available, the process will not run.**
+
+### Coherence
+The phase measurements in the two images used in InSAR must be coherent in order to detect change. Random changes in phase from one acquisition to the next can mask actual surface deformation. Vegetation is a common driver of decorrelation, as changes can easily take place in the interval between two acquisitions due to growth, seasonal changes, or wind effects. It will be difficult to generate valid interferograms with C-band data in heavily vegetated regions due to lack of coherence even with fairly short time intervals.
+
+Consider seasonality when selecting image pairs. Decorrelation can be particularly high when comparing phase from different seasons. Changes in the condition of vegetation (especially deciduous canopies), snow, moisture, or freeze/thaw state can impact phase measurements. In cases where a temporal baseline is required that spans seasons, it may be better to use an annual interferogram if possible so that the images are more comparable in terms of seasonality.
+
+### Line-of-Sight Measurements
+When looking at a single interferogram, the deformation measurements in the line-of-sight orientation of the sensor indicate relative motion towards or away from the sensor. InSAR is not sensitive to motion in the azimuth direction of the satellite, so motion that occurs in the same direction as the satellite's direction of travel will not be detected.
+
+A single interferogram cannot be used to determine the relative contributions of vertical and horizontal movement to the line-of-sight displacement measurement. The vertical displacement map is generated based on the assumption that the movement is entirely in the vertical direction, which may not be realistic for some processes. To determine how much of the signal is driven by vertical vs. horizontal movement, you must either use a time series of interferograms, or use reference measurements with known vertical and horizontal components (such as GNSS measurements from the region of deformation) to deconstruct the line-of-sight displacement.
+
+All displacement values are calculated relative to a [reference point](#phase-unwrapping-reference-point "Jump to Phase Unwrapping Reference Point part of Limitations section in this document"), which may or may not be an appropriate benchmark for measuring particular areas of displacement within the interferogram.
+
+### Phase Unwrapping Reference Point
```

Comment:
Does this need to be amended for the burst products? Do we even include LOS and vertical displacement maps? How is the reference point selected for the burst InSAR products?

Reply:
No, we don't (currently) include displacement maps for burst InSAR products. The output files are:
```
.txt  
_unw_phase.tif  
_README.md.txt  
_lv_theta.tif  
_lv_phi.tif  
_corr.tif  
_wrapped_phase.tif  
_unw_phase.png  
_water_mask.tif  
_dem.tif  
_conncomp.tif  
```

I'm not sure on the reference point. ISCE uses [snaphu](http://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/snaphu_man1.txt) to do the unwrapping, and I haven't found any mention of a reference point in the manual.

Reply:
Here's an interesting discussion of the reference point used in topsApp:
https://github.com/isce-framework/isce2/discussions/464

Reply:
Regardless, we should probably pull out some of those errors/limitations sections from the common template. I'll take a closer look tomorrow.